PIL/Image.py

```diff
@@ -1426,6 +1496,7 @@ def alpha_composite(self, im, dest=(0,0), source=(0,0)):
         self.paste(result, box)
 
     def point(self, lut, mode=None):
+        # type: (Union[List, Callable[[Any], Any]], Optional[Mode]) -> Image
```

Comment:
Also that's a `List[float]`?
We can just use float here, since both are possible?